backend/substrapp/tasks/tasks.py

```diff
@@ -581,10 +583,10 @@ def _do_task(client, subtuple_directory, tuple_type, subtuple, compute_plan_id,
     # local volume for train like tuples in compute plan
     if compute_plan_id is not None and tuple_type != TESTTUPLE_TYPE:
         volume_id = f'local-{compute_plan_id}-{org_name}'
-        if rank == 0:
-            client.volumes.create(name=volume_id)
-        else:
+        try:
```

Comment:
The behavior is changed here, local volume should only be created on rank == 0

Reply:
Thanks for asking, I wasn't 100% sure regarding this.

It has been confirmed with @camillemarini, in case of a compute plan, local volume will be created if it does not exist whatever the rank value.